compressor/storage.py

```diff
@@ -85,6 +86,37 @@ def save(self, filename, content):
         return filename
 
 
+class BrotliCompressorFileStorage(CompressorFileStorage):
+    """
+    The standard compressor file system storage that gzips storage files
+    additionally to the usual files.
+    """
+    chunk_size = 1024
```

Comment:
is this really necessary? i mean, we are talking about a few megabytes of data at the maximum, aren't we? were you running into any memory usage issues?